### PR TITLE
Add playwright testing

### PR DESCRIPTION
**Pull request by cdrake**
Adds playwright testing 

- Install and configure playwright
- Add test to playwright/e2e
- Generate initial snapshot
- Added playwright github action

### CODE REVIEW

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
```
Comment (jens-ox) [inline .github/workflows/playwright.yml:12]: Current major version is v4

File: .github/workflows/playwright.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    - uses: actions/setup-node@v3
```
Comment (jens-ox) [inline .github/workflows/playwright.yml:13]: Current major version is v4

File: .github/workflows/playwright.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      with:
+        node-version: 18
```
Comment (jens-ox) [inline .github/workflows/playwright.yml:15]: Any reason for not using the current LTS version (20)?

File: package.json [hunk 4]
```diff
@@ -13,16 +13,18 @@
   },
   "scripts": {
     "dev": "vite",
-    "build": "npx prettier --write src/ && npx vite build --config vite.config.js --emptyOutDir --base=./ && vite build --config vite.config_inject.js --emptyOutDir --base=./ && npm run copy-test && npm run copy-demo",
+    "build": "npx prettier --write src/ && npx vite build --config vite.config.js --emptyOutDir --base=./ && vite build --config vite.config_inject.js --emptyOutDir --base=./ && npm run copy-test && npm run copy-demo && npm run copy-test-playwright",
```
Comment (jens-ox) [inline package.json:16]: As this is not really related to building the library, it might make sense to move this step to `test-playwright`, see next comment

File: package.json [hunk 5]
```diff
@@ -13,16 +13,18 @@
...
     "demo": "npm run build && npx http-server demos/ --cors",
     "demo-win": "npm run build-win && npx http-server demos/",
     "copy-test": "cp ./dist/niivue.umd.js ./tests/niivue.umd.js",
     "copy-test-win": "copy .\\dist\\niivue.umd.js .\\tests\\niivue.umd.js",
+    "copy-test-playwright": "cp ./dist/niivue.umd.js ./playwright/e2e/niivue.umd.js",
     "copy-demo": "cp -r ./dist/* ./demos/features",
     "copy-demo-win": "copy .\\dist\\niivue.umd.js .\\demos\\features\\niivue.umd.js",
     "test": "npm run build && jest --maxWorkers=1",
     "test-win": "npm run build-win && jest --maxWorkers=1",
+    "test-playwright": "npm run build && npx playwright test",
```
Comment (jens-ox) [inline package.json:27]: I think it might make more sense to do `npm run build && npm run copy-test-playwright && npx playwright test` and remove `npm run copy-test-playwright` from `build`

File: package.json [hunk 6]
```diff
@@ -58,7 +60,9 @@
     "uuid": "^9.0.0"
   },
   "devDependencies": {
+    "@playwright/test": "^1.40.0",
     "@rollup/plugin-commonjs": "^24.0.1",
+    "@types/node": "^20.9.2",
```
Comment (jens-ox) [inline package.json:65]: This (and `@ts-check` in `playwright.config.js`) is probably not really needed right now, as TS won't run without a `tsconfig.json`.

**Review by hanayik — APPROVED** [review]
